src/components/vm/disks/diskAdd.jsx

```diff
@@ -64,7 +64,23 @@ function getFilteredVolumes(vmStoragePool, disks) {
     return filteredVolumesSorted;
 }
 
-const SelectExistingVolume = ({ idPrefix, storagePoolName, existingVolumeName, onValueChanged, vmStoragePools, vmDisks }) => {
+function getDiskUsageMessage(vms, storagePool, volumeName) {
+    const isVolumeUsed = getStorageVolumesUsage(vms, storagePool);
+
+    if (!isVolumeUsed[volumeName] || (isVolumeUsed[volumeName].length === 0))
+        return null;
+
+    const vmsUsing = isVolumeUsed[volumeName].join(', ');
+    const volume = storagePool.volumes.find(vol => vol.name === volumeName);
+
+    let message = cockpit.format(_("This volume is already used by $0. "), vmsUsing);
```

Comment:
Please no leading or trailing spaces, they are almost guaranteed to get lost in translation. Please add the " " separately (and not translated)